taxonomy-editor-frontend/src/pages/go-to-project/index.tsx

```diff
@@ -8,34 +8,54 @@ import EditIcon from "@mui/icons-material/Edit";
 import useFetch from "../../components/useFetch";
 import { API_URL } from "../../constants";
 import { toSnakeCase, toTitleCase } from "../../utils";
+import type { ProjectsAPIResponse } from "../../backend-types/types";
 
-const GotoProject = ({ resetNavLinks }) => {
-  const {
-    data: incomingData,
-    isPending,
-    isError,
-    errorMessage,
-  } = useFetch(`${API_URL}projects?status=OPEN`);
-  const [projectData, setProjectData] = useState([]);
+type ProjectType = {
+  id: string;
+  projectName: string;
+  taxonomyName: string;
+  branchName: string;
+  description: string;
+  errors_count: number;
+};
```

Comment:
Would it be better if we stored this in a separate `frontend-types/types.ts` folder?

Reply:
think is ok to keep it in here, it's better to keep things as nearest to the place they are used. In this case for the moment it will be used only here. For `backend-types` does make sense to have a single source of thruth for the types as those affect all the frontend and not only parts of it.